scripts/get_hi-mia_data.py

```diff
@@ -0,0 +1,152 @@
+# Copyright (c) 2019 NVIDIA Corporation
+#
+# USAGE: python get_aishell_data.py --data_root=<where to put data>
+
+import argparse
+import json
+import logging
+import os
+import tarfile
+import urllib.request
+from glob import glob
+
+import librosa as l
+from sklearn.model_selection import StratifiedShuffleSplit
```

Comment:
do we have sklearn in our requirements?

Reply:
sklearn gets installed from one of our dependencies not sure which one

Reply:
Can you add it to requirements_asr.txt to be sure?

Reply:
Actually it's in requirements_nlp, can you move it to requirements

Reply:
Done